### PR TITLE
Hikey: Add flash and nvme targets

### PR DESCRIPTION
**Pull request by jbech-linaro**
Adds two new targets, one that downloads the necessary nvme.img file and
another target that flash the device. We have also added a couple of
.PHONY directives to a few targets.

Signed-off-by: Joakim Bech <joakim.bech@linaro.org>

### CODE REVIEW

**Comment by vchong** [comment]
```Reviewed-by: Victor Chong <victor.chong@linaro.org>```

File: hikey.mk [hunk 1]
```diff
@@ -337,3 +358,24 @@ lloader-clean:
...
 	fi
+
+################################################################################
+# nvme image
+################################################################################
+.PHONY: nvme
+nvme:
+	wget https://builds.96boards.org/releases/hikey/linaro/binaries/latest/nvme.img -O $(NVME_IMG)
+
+nvme-cleaner:
```
Comment (pascal-brand-st-dev) [inline hikey.mk]: Missing PHONY on nvme-cleaner

**Comment by pascal-brand-st-dev** [comment]
Apart a minor comment;

    Reviewed-by: Pascal Brand <pascal.brand@linaro.org>

**Comment by jbech-linaro** [comment]
Comments addressed, squashed and tags applied, ready to be merged.

**Pull request merged**